pom.xml

```diff
@@ -111,5 +141,23 @@
             <version>2.21.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>2.9.4</version>
```

Comment:
2.9.9 is current. 2.9.4 may be susceptible to that Jackson recent bug.  Only test scope, but best to keep fresh